.github/workflows/release.yaml

```
@@ -0,0 +1,27 @@
+name: Release
+
+# Run workflow on commits to the `master` branch
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Install Java and Maven
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

````suggestion
      - name: Install Java and Maven
        uses: actions/setup-java@v3.3.0
        with:
          distribution: 'zulu'
          java-version: '11'
````

We should use the same version of the action all workflows